test/unit/initialize.spec.ts

```diff
@@ -669,4 +670,30 @@ describe("Initialization", () => {
       expect(lastPanel.next().getPosition() - (lastPanel.getPosition() + lastPanel.getSize())).equals(gap);
     });
   });
+  describe("initialize component with decorator", () => {
+    it("should check if the method of the class created with the decorator is properly entered.", () => {
+      @withFlickingMethods
+      class TestFlicking {
+        private flicking: Flicking;
+        constructor() {
+          const flickingInfo = createFlicking(horizontal.shouldClone4, {
+            gap: 10,
+            anchor: "0",
+            hanger: "0",
+            circular: false,
+            defaultIndex: 0,
+          });
+          this.flicking = flickingInfo.instance;
+        }
+      }
+      const flicking: any = new TestFlicking();
+
+      expect(flicking.sync).to.be.not.ok;
+      expect(flicking.getCloneCount).to.be.not.ok;
+      expect(flicking.append).to.be.not.ok;
+      expect(flicking.prepend).to.be.not.ok;
```

Comment:
It's best not to use `ok`, even chai's own guide says that.
<img width="390" alt="스크린샷 2019-05-22 오전 9 34 20" src="https://user-images.githubusercontent.com/26213435/58139231-df8b2700-7c74-11e9-9838-35a5883e1d55.png">
https://www.chaijs.com/api/bdd/#method_ok